src/openforms/appointments/tokens.py

```diff
@@ -0,0 +1,92 @@
+from datetime import date
+
+from django.conf import settings
+from django.utils import timezone
+from django.utils.crypto import constant_time_compare, salted_hmac
+from django.utils.http import base36_to_int, int_to_base36
+
+from openforms.submissions.models import Submission
+
+
+class SubmissionAppointmentTokenGenerator:
```

Comment:
#655 is still pending, but it has a refactor for token generators in general. Depending on which PR gets merged first, the other will have to adapt to make use of this.

Reply:
Okay good to know.  I'm guessing you mean once #655 is merged that this will need to inherit from `BaseTokenGenerator`?

Reply:
correct, and it's merged now :)